health/health.d/geth.conf

```diff
@@ -0,0 +1,17 @@
+template: chainhead_diff_between_header_block
+on: geth.chainhead
+every: 5s
```

Comment:
Hello @OdysLam , as you can see [here](https://github.com/netdata/netdata/blob/master/health/health.d/backend.conf#L4-L6) we inserted new components for our alarms, and this template does not have them, please, can you add for us?

Reply:
Also, consider formatting the alarms like

https://github.com/netdata/netdata/blob/03002eb22812ca342a591e9b961821e2c0afdcd1/health/health.d/adaptec_raid.conf#L4-L15

Reply:
Still _invalid_ formatting. If you check other alarms you notice that we don't do _align left_, but _align center_.